sr_hand/scripts/sr_hand/tactile_receiver.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, prefix=""):
         """
         # appends trailing slash if necessary
         if not prefix.endswith("/"):
+            #kind of a hack to remove trailing underscores
+            if prefix.endswith("_"):
```

Comment:
not the prettiest implementation but I couldn't think of something simple in that case.